### ISSUE TITLE
Branch [Development] Has Died

### ISSUE THREAD

**Issue by DanielAirgood** [LocanCollado/EnterpriseFinal#23]
After Creating the Pre-Alpha 0.1 version of the project the code in the Development branch has become so outdated, that it is now impossible to merge with any version of the code after Pre-Alpha 0.1

to call the development branch dead is an understatement... It's been burned at the stake, then they burned the ashes, then THOSE ashes have been nuked from orbit, and salted the earth in a 500 mile radius just to be safe.

This is what happened:
I wasn't able to just add a website to the project, so I had to start an entirely new project as a website. AND I wasn't able to add our existing code to THAT project, so I had to make ANOTHER project as a class library and add all of our code to the that project. THEN I had to add the Web project to Class library project and then edit a bunch of Visual Studio settings and properties to recognize everything. All said and done, the project was now too big to upload into github. So, I had to fork the project, delete every file one by one, download the github desktop app, and upload the project into it as a new repository, copy those files over to the fork, add the entire project to the fork as one gigantic commit, make a new branch in the original fork and then copy and paste everything into it since there were too many changes to merge, AND FINALLY!!! open up gitshell and manually merge the branch with the master... with a command line editor... one line at a time.... for every conflicting line.... all 600 of them....

TL:DR DONT FUCKING TOUCH THE DEVELOPMENT BRANCH UNLESS YOU HAVE A DEATH WISH!!!
